js/foundation.drilldown.js

```diff
@@ -85,7 +85,11 @@ class Drilldown {
       var $menu = $(this),
           $back = $menu.find('.js-drilldown-back');
       if(!$back.length){
-        $menu.prepend(_this.options.backButton);
+        if(_this.options.backButtonPosition == 'bottom'){
+          $menu.append(_this.options.backButton);
+        } else {
+          $menu.prepend(_this.options.backButton);
+        }
```

Comment:
Well, we *are defining* the default value (l. 406), so I don't see how this option could be `unset`. We are not introducing a breaking change. Websites that are currently using the drilldown menu will still have the back button at the top, unless they overwrite `backButtonPosition` with the `bottom` value. An error would be throw only if they overwrite it with a value different from `top` and `bottom`.